voyager/plugins/transaction/aptos/src/main.rs

```diff
@@ -236,16 +236,16 @@ impl PluginServer<ModuleCall, ModuleCallback> for Module {
                             let signed_tx = raw.sign(pk, pk.public_key()).unwrap();
 
                             dbg!(&signed_tx);
+                            let res = self
+                                .aptos_client
+                                .submit(&signed_tx.clone().into_inner())
+                                .await
+                                .unwrap();
```

Comment:
we're doing each message individually now?

Reply:
Movement for some reason can't handle batch transactions for now, will have a comment why we are doing this. I asked but couldnt get a response, will ask again after the integration is finished.